lifelines/fitters/aalen_johansen_fitter.py

```diff
@@ -0,0 +1,174 @@
+from __future__ import print_function
+from __future__ import division
+import numpy as np
+import pandas as pd
+import warnings
+
+from lifelines.fitters import UnivariateFitter
+from lifelines.utils import _preprocess_inputs, inv_normal_cdf
+from lifelines.fitters.kaplan_meier_fitter import KaplanMeierFitter
+
+class AalenJohansenFitter(UnivariateFitter):
+    """
+    Class for fitting the Aalen-Johansen estimate for the cumulative incidence function in a competing risks framework.
+    Treating competing risks as censoring can result in over-estimated cumulative density functions. Using the Kaplan
+    Meier estimator with competing risks as censored is akin to estimating the cumulative density if all competing risks
+    had been prevented. If you are interested in learning more, I (Paul Zivich) recommend the following open-access
+    paper; Edwards JK, Hester LL, Gokhale M, Lesko CR. Methodologic Issues When Estimating Risks in
+    Pharmacoepidemiology. Curr Epidemiol Rep. 2016;3(4):285-296.
+    
+    AalenJohansenFitter(alpha=0.95,jitter_level=0.00001,seed=None)
+    
+    Aalen-Johansen cannot deal with tied times. We can get around this by randomy jittering the event times 
+    slightly. This will be done automatically and generate a warning.
+    """
+    def __init__(self, jitter_level=0.0001, seed=None, alpha=0.95):
+        UnivariateFitter.__init__(self, alpha=alpha)
+        self._jitter_level = jitter_level
+        self._seed = seed  # Seed is for the jittering process
+
+    def fit(self,durations, event_observed, event_indicator, timeline=None, entry=None, label='AJ_estimate', 
+            alpha=None, ci_labels=None, weights=None):
+        """
+        Parameters:
+          duration: an array or pd.Series of length n -- duration of subject was observed for 
+          event_observed: an array, or pd.Series, of length n. Integer indicator of distinct events. Must be 
```

Comment:
this is a neat generalization of how I usually use `event_observed` - fits nicely.